docs/hosting/configuration/environment-variables/logs.md

```diff
@@ -28,6 +28,7 @@ This page lists environment variables to set up logging for debugging. Refer to
 | `DB_LOGGING_ENABLED` | Boolean | `false` | Whether to enable database-specific logging. |
 | `DB_LOGGING_OPTIONS` | Enum string: `query`, `error`, `schema`, `warn`, `info`, `log`  | `error` | Database log output level. To enable all logging, specify `all`. Refer to [TypeORM logging options](https://orkhan.gitbook.io/typeorm/docs/logging#logging-options){:target=_blank .external-link} |
 | `DB_LOGGING_MAX_EXECUTION_TIME` | Number | `1000` | Maximum execution time (in milliseconds) before n8n logs a warning. Set to `0` to disable long running query warning. |
+| `CODE_ENABLE_STDOUT` | Boolean | `false` | Set to`true` to send Code node logs to process's stdout. |
```

Comment:
```suggestion
| `CODE_ENABLE_STDOUT` | Boolean | `false` | Set to `true` to send Code node logs to process's stdout. |
```

Reply:
Can we explain the circumstances when this is useful?

Reply:
So something like "Set to`true` to send Code node logs to process's stdout for debugging, monitoring, or logging purposes. Recommended only in non-production environments for security reasons." 

Should I include the second sentence as we disabled this by default for a reason stated [here](https://community.n8n.io/t/n8n-ignores-n8n-log-level-env-var-value/10916/15)? What do you think?

Reply:
That sounds good